napari/_tests/test_status.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+from napari.utils.status_messages import status_format
+
+
+def test_status_format(make_napari_viewer):
+    """ test various formatting cases embodied in utils.status_messages.status_format """
+
+    values = np.array(
```

Comment:
Can you also make this test look more like https://github.com/napari/napari/blob/133ab7f70803b7db26f772335b30fb96e4a44b02/napari/utils/_tests/test_misc.py#L21-L35 where we have `input, expected` and then the test just becomes `assert status_format(input) == expected`

Reply:
thanks, changes made!